source/contracts/reporting/Universe.sol

```diff
@@ -267,27 +292,37 @@ contract Universe is DelegationTarget, Extractable, ITyped, Initializable, IUniv
     }
 
     function getValidityBond() public onlyInGoodTimes returns (uint256) {
```

Comment:
Perhaps this should be called something like `setAndGetValidityBond` and `getValidityBondView` should be called `getValidityBond`?

Reply:
Yea that's a good idea. Will do

Reply:
Same with the target gas getter